### PR TITLE
co2 SM a bit more dangerous

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Increase the CO2 SM heat penalty to 6

## Why It's Good For The Game
CO2 SMs are too easy to set and forget, we should strive to have only the basic N2 setup to be set and forget or to have an actual effort to reach a mix and a setup that is basically set and forget (like plasma SM with a dynamic pressure release that you build)

## Changelog

:cl:
balance: Increase the CO2 SM heat penalty to 6
/:cl:

### CODE REVIEW

**Comment by daplanman12** [comment]
Yet another change that is unnecessary and only serves to make the game worse by forcing engineering to play how Ghliker thinks is correct. CO2 SMs are already more fragile and vulnerable to sabotage as a result of being closer to delam levels, no need to make them need to be babied so that players won't try to experiment with them.

**Comment by Pickle-Coding** [comment]
Currently standard SM (replacing useless pumps only) can easily handle CO2 up to 5GeV, and near 7GeV if you add protection against anomalies. It having a heat penalty of 0.1 (it gets clamped to a minimum of 0.5) makes it produce the same heat as an N2 engine per eV, and has a higher specific heat than N2, making it a better coolant. It is much more stable than just adding extra emitters for even less effort.

You wouldn't need an extreme engine for this change unless you're planning to get above 9GeV, it will just make the roundstart setup not handle it so easily.

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by FernandoJ8** [comment]
Good change, CO2s are way too stable and easy for how efficient they are

File: code/__DEFINES/supermatter.dm [hunk 1]
```diff
@@ -30,6 +30,7 @@
 #define N2O_HEAT_RESISTANCE 6          //Higher == Gas makes the crystal more resistant against heat damage.
 #define HYDROGEN_HEAT_RESISTANCE 2 // just a bit of heat resistance to spice it up
 #define PROTO_NITRATE_HEAT_RESISTANCE 5
+#define CO2_HEAT_RESISTANCE -2
```
[the remark this answers is not in the document]
Comment (Ghilker) [reply code/__DEFINES/supermatter.dm:33]: i made some changes and the number will not go under 0. 
the SM doesn't take extra damage on something like 4500 MeV (the basic range of a CO2 SM)

[the remark this answers is not in the document]
Comment (LemonInTheDark) [reply code/__DEFINES/supermatter.dm:33]: If you want to disincentivize co2 sms why not tweak the things that already exist?

**Comment by Coastfront** [comment]
Considering CO2 is meant to be one of the roundstart gasses that is difficult to run a sm on (so far the only super challenging ones is stuff like trit and plasma) yeah this is good.

**Comment by LemonInTheDark** [comment]
Fuck it

**Comment by KareRott** [comment]
This change does not add fun. I understand that you want to encourage people to make more interesting setups. But in reality, all this change really does is keep people from wanting to branch out from n2. This does not promote diversity.

However, I've a suggestion that might make this more interesting. Keep the heat value change but increase the co2 consumption and increase the potential plouxium output.  (I can imagine You'd still have to use other gasses and a very robust cooling setup in order to run such a loop. the crystal may even be on fire half the time because of the new heat penalty.)

There are many an atmos tech who dream of replacing the station's o2 with plouxium, but the task of such is absolutely gargantuan, and still would be with such a change. However, this would at the very least preserve the spirit and promote more diverse SM builds.

**Comment by KareRott** [comment]
Oh uh, pure co2 setups are currently impossible by the way, not sure if that was really intended but you should really test a pure co2 loop with this current change to see what I mean. Like, if you intended to make it produce the same amount of eVs as an n2 setup, it just won't without delaming.

**Pull request merged**